src/remote.c

```diff
@@ -149,6 +149,51 @@ dcc_send_header(int net_fd,
     return 0;
 }
 
+/**
+ * Check if the preprocessed source depends on local file(s)
+ *
+ * @param cpp_fname preprocessed file to check
+ * @param input_fname corresponding source file name
+ *
+ * @return 0 if no local dependencies found, 1 otherwise
+ *
+ * For now checks for assembler '.incbin' directive, which could be
+ * part of inline assembly
+ */
+static int
+dcc_check_unsupported_directives(const char *cpp_fname, const char *input_fname)
+{
+    FILE *cpp_f = NULL;
+    char *line = NULL;
+    size_t len = 0;
+    ssize_t bytes_read = 0;
+    int ret = 0;
+
+    cpp_f = fopen(cpp_fname, "r");
+    if (!cpp_f) {
+        rs_log_warning("failed to open preprocessed source %s", input_fname);
+	goto out;
+    }
+
+    errno = 0;
```

Comment:
I feel like assigning to errno is considered bad a bad idea but I see it's already common in this tree, so, OK.

Reply:
It's not necessary, will remove it.